### ISSUE TITLE
Problem when binding options Got an error when using my-date-picker component.

### ISSUE THREAD

**Issue by redwolfgang24** [kekeh/mydatepicker#426]
Can't bind to 'options' since it isn't a known property of 'my-date-picker'.
1. If 'my-date-picker' is an Angular component and it has 'options' input, then verify that it is part of this module.
2. If 'my-date-picker' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.
3. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component. ("
							<my-date-picker
								name="birthDate"
								[ERROR ->][options]="myDatePickerOptions"
								formControlName="birthDate"
								(dateChanged)="onDateChan"): ng:///ComponentsModule/ProfileViewComponent.html@41:8
'my-date-picker' is not a known element:

in my `package.json` I have this
```
"devDependencies": {
    "mydatepicker": "~1.2.1"
}
```

in my app.module.ts I have this
```
import { MyDatePickerModule } from 'mydatepicker';

@NgModule({
	imports: [
		BrowserModule,
		FormsModule,
		ComponentsModule,
		MyDatePickerModule
	],
	bootstrap: [
		AppComponent
	],
	schemas: [
		CUSTOM_ELEMENTS_SCHEMA
	],
})
export class AppModule {
}
```

Can you help me about this one @kekeh ? I think I miss something.
Thank you in advance @kekeh :)

**Comment by kekeh** [comment]
Difficult to say what causes the error, but can you update the latest version 2.0.31.

**Comment by redwolfgang24** [comment]
I already updated to the latest version but still got the same issues. :(

**Comment by redwolfgang24** [comment]
This is my options
```
myDatePickerOptions: IMyDpOptions = {
	todayBtnTxt: 'Today',
	dateFormat: 'yyyy-mm-dd',
	firstDayOfWeek: 'mo',
	sunHighlight: true,
	inline: false,
	disableUntil: {
		year: 2016, month: 8, day: 10
	}
};
```
And this one how I call the component
```
<my-date-picker
	[options]="myDatePickerOptions"
	formControlName="birthDate"
	(dateChanged)="onDateChanged($event)"></my-date-picker>
```

I don't know whats wrong @kekeh.

**Comment by kekeh** [comment]
Your configuration seems to be ok. You don't need to set CUSTOM_ELEMENTS_SCHEMA in @NgNodule.

I have tested component many versions of @angular/cli and I haven't seen this kind of error. Are you using @angular/cli?

Is this build or runtime error?

**Comment by redwolfgang24** [comment]
a runtime error. I saw the error in the console.

**Comment by kekeh** [comment]
Did you follow instructions from Here: https://github.com/kekeh/mydatepicker#installation

**Comment by redwolfgang24** [comment]
Yes, I follow all the instructions but I still got the same error. :(

**Comment by teslavitas** [comment]
Hi, I have the same error message when I try to embed my-date-picker to a component. If I use it as a part of the page - it works.

**Comment by kamlekar** [comment]
Hi @kekeh, I am facing the same issue here. commented here: https://github.com/kekeh/mydatepicker/issues/288#issuecomment-343872817

Please help.

**Comment by aidionidis** [comment]
I replied in #288, check if that helps you.

**Comment by sailendebnath** [comment]
Hi @kekeh, I am facing the same issue 
![capture](https://user-images.githubusercontent.com/16538324/47010648-3d92e180-d15d-11e8-8180-58ae946dd6ac.PNG)

could you please help me to resolve this

**Comment by nitin123gupta** [comment]
Is there any solution for this??

**Comment by jsbimra** [comment]
I am facing this too, is anybody found fix related to this? @kekeh
